packages/transport/test/end2end/End2End.test.ts

```diff
@@ -455,6 +455,357 @@ describe("RelationshipTest: Terminate", function () {
     });
 });
 
+describe("RelationshipTest: Accept Reactivation", function () {
+    let connection: IDatabaseConnection;
+    let transport: Transport;
+
+    let from: AccountController;
+    let to: AccountController;
+    let relationshipId: CoreId;
+
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+        transport = TestUtil.createTransport(connection);
+
+        await transport.init();
+
+        const accounts = await TestUtil.provideAccounts(transport, 2);
+        from = accounts[0];
+        to = accounts[1];
+        relationshipId = (await TestUtil.addRelationship(from, to)).acceptedRelationshipFromSelf.id;
+        await from.relationships.terminate(relationshipId);
+        await TestUtil.syncUntilHasRelationships(to);
+    });
+
+    afterAll(async function () {
+        await from.close();
+        await to.close();
+        await connection.close();
+    });
+
+    test("should request reactivating a relationship between two accounts and accept the reactivation", async function () {
+        await from.relationships.reactivate(relationshipId);
+        await TestUtil.syncUntilHasRelationships(to);
```

Comment:
Again, I'd expect a separate test for `reactivate` (matching auditLog is created).